crdppf/static/js/Crdppf/featurePanel.js

```diff
@@ -0,0 +1,278 @@
+                                        singleClickExpand: true,
+                                        attributes: jsonData[j],
+                                        text: Crdppf.labels.restrictionFoundTxt + (j+1) + ' : ' + String(jsonData[j].data.intersectionMeasure),
+                                        draggable: false,
+                                        leaf: false,
+                                        expanded: false,
+                                        id: Crdppf.uuid(),
+                                        listeners: {
+                                            'click': function(node,e) {
+                                                Crdppf.Map.intersectLayer.removeAllFeatures();
+                                                feature = node.attributes.attributes;
+                                                Crdppf.Map.intersectLayer.addFeatures(feature);
+                                                Crdppf.Map.map.zoomToExtent(feature.geometry.bounds);
+                                            }
+                                        }
+                                    });
+                                    // create node containing the feature attributes (level 3)
+                                    var contentNode = new Ext.tree.TreeNode({
+                                        cls: 'contentNodeCls',
+                                        text: html,
+                                        draggable: false,
+                                        leaf: false,
+                                        expanded: false,
+                                        id: Crdppf.uuid()
+                                    });
+                                    sameLayerNode.appendChild(contentNode);
+                                    layerChild.appendChild(sameLayerNode);
+                                }
+                                Crdppf.FeaturePanel.root.appendChild(layerChild);
+                            }
+                            lList.push(lName);
+                        }
+                    }
+                    if (jsonData.length === 0){
+                        // create layer node (level 1, root is level 0 in the hierarchy)
+                        var layerChild =  new Ext.tree.TreeNode({
+                            text: Crdppf.labels.noRestrictionFoundTxt,
+                            draggable: false,
+                            id: Crdppf.uuid(),
+                            leaf: false,
+                            expanded: true
+                        });
+                        Crdppf.FeaturePanel.root.appendChild(layerChild);
+                    }
+            }
+            // define an request object to the interection route
+            var featureMask = new Ext.LoadMask(this.featureTree.body, {msg: Crdppf.labels.restrictionLoadingMsg});
+            featureMask.show();
+
+            var request = OpenLayers.Request.GET({
```

Comment:
`Openlayer.Request.GET` fires a request, right ?

If yes, why do you need do declare it with a variable (moreover I cannot see anywhere this one recalled).

Couldn't you just do:
```javascript
OpenLayers.Request.GET({
    ....
});
```